h/feeds/util.py

```diff
@@ -0,0 +1,18 @@
+"""Utility functions for feed-generating code."""
+import urlparse
+from h import util
+
+
+def tag_uri_for_annotation(annotation, annotation_url):
+    """Return a tag URI (unique identifier) for the given annotation.
+
+    Suitable for use as the value of the <id> element of an <entry> in an
+    Atom feed, or the <guid> element of an <item> in an RSS feed.
+
+    :returns: A tag URI (RFC 4151) for use as the ID for the Atom entry.
+    :rtype: string
+
+    """
+    return u"tag:{domain},{day}:{id_}".format(
+        domain=urlparse.urlparse(annotation_url(annotation)).netloc,
+        day=annotation.created_day_string, id_=annotation["id"])
```

Comment:
The date in an RSS tag is the date at which the tag format was first used by the RSS generator, not the date the annotation was created. I'd suggest just shoving a constant in this file with today's date in it and using that.